app/src/main/java/br/com/alexf/boraprofut/features/playersForm/PlayersFormScreen.kt

```diff
@@ -41,152 +49,208 @@ import br.com.alexf.boraprofut.ui.theme.ClearPlayersTextFieldContainerColor
 import br.com.alexf.boraprofut.ui.theme.DuplicatesNamesContainerColor
 import br.com.alexf.boraprofut.ui.theme.SavePlayersButtonContainerColor
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun PlayersFormScreen(
     uiState: PlayersUiState,
     onSavePlayers: () -> Unit,
     onClearPlayers: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier
-            .fillMaxSize()
-            .verticalScroll(rememberScrollState())
-            .background(MaterialTheme.colorScheme.background)
-    ) {
+    val isToolTipVisible = uiState.isToolTipVisible.collectAsState(initial = true)
+    Scaffold(

```

Comment:
Tudo bem de usar o `Scaffold`, só vale um ponto de atenção se a ideia dele for usar exclusivamente um slot! Nesses casos, talvez nem compense usá-lo.

Reply:
Acredito que fica melhor para colocar os botões de ação e também para quando houver rolagem da tela manter o título